### ISSUE TITLE
switch from xf86-input-nextwindow to evdev input for multitouch

### ISSUE THREAD

**Issue by tinkermaniac** [glorang/nwfermi#14]
how to switch from xf86-input-nextwindow to evdev input ? because it supports one onw touch but evdev supports multitouch. xf86-input-nextwindow supports only single touch but in dmonad/nwfermi it's mentioned that avdec recognises more than one fingers, i followed your tutorial, how can i disable xf86-input-nextwindow and replace it with evdev ?

**Comment by glorang** [comment]
I don't think this will work but you can give it a try.

Easiest would be to edit `/etc/X11/xorg.conf.d/10-nwfermi.conf` and change the Driver line to `Driver "evdev"` and check your xorg log to check what driver is in use.

**Comment by tinkermaniac** [comment]
someone shares this patched version supports multitouch with touchegg daemon : https://lirein.org/_downloads/nwfermi/

I will try it out today by a complete reinstall and later report if it's working... [I am currently on linux mint 21.3]

**Comment by rmeissn** [comment]
@tinkermaniac Did you had any success in getting multitouch working?

**Comment by tinkermaniac** [comment]
nope, it didn't work ;-;  . I tried different versions, the one which was on launchpad (labelled as multitouch) didn't work too. Though I found a fix for windows, a drivers by medion corporation support multitouch. Some small changes are required in the config if it's not one of the supported screens or a part of median akoya series.
